site/content/en/latest/tasks/extensibility/envoy-patch-policy.md

```diff
@@ -82,6 +82,89 @@ data:
 
 {{< boilerplate rollout-envoy-gateway >}}
 
+## XDS Name Scheme V2
+
+Starting from v1.5, Envoy Gateway uses version 2 of the xDS name scheme when generating xDS resources.
+Because [EnvoyPatchPolicy][] relies on specific xDS resource names, it’s important to use the correct naming format when authoring a patch policy.
+
+| Component            | Scheme Version | Format Description                                                          | Example                          |
+|----------------------|----------------|------------------------------------------------------------------------------|----------------------------------|
+| **Listener name**    | Old            | `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`                    | `default/eg/http`               |
+|                      | V2             | `<Protocol>-<Port>`                                                         | `tcp-80`                         |
+| **RouteConfig name** | Old            | `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`                    | `default/eg/http`               |
+|                      | V2 (HTTP)      | `http-<Port>`                                                               | `http-80`                       |
+|                      | V2 (HTTPS)     | `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`                    | `default/eg/https`               |
```

Comment:
Actually we need to apply the same name pattern of HTTP to HTTP3 - this will be addressed in a follow-up PR.